templates/partners/request-login.html

```diff
@@ -154,4 +154,4 @@ <h1>Find a partner</h1>
       }
     </style>
 
-    {% endblock %}
+    {% endblock %}
```

Comment:
Is this change needed?

Reply:
It wasn't intentional, I'd originally removed this page but I had to add it back. Extra line got added when I copy/pasted it back in